### DISCUSSION TITLE
Metodo de ordenamiento inplace?

### DISCUSSION THREAD

**Discussion by Burbina200** [IIC2133-PUC/2024-1#50] [Tarea 0] [answered]
Hola! Tengo una consulta respecto al ordenamiento utilizado en el comando "REPORTE-PLANETAS". ¿Es necesario que este ordenamiento se realice de manera inplace? Actualmente, no he implementado el ordenamiento inplace debido a un inconveniente relacionado con los IDs de los planetas en los pedidos.

**Comment by PabloNavarroY** [comment, marked as answer]
Tienes total libertad para realizar la tarea como te dé la gana. Lo único que importa es el resultado y que aprendas a ocupar C.

**Reply by jnbaltra** [reply]
se puede usar una funcion qsort?
